### PR TITLE
Mops and crematoriums now call parent on Destroy

### PR DESCRIPTION
**Pull request by MadmanMartian**
After watching an indestructable mop and crematorium orbit the singularity, it sadly had to be done

### CODE REVIEW

**Comment by ShiftyRail** [comment]
Ah, fuck I was doing it
Maybe `. = ..()` though I don't think we care what `Destroy()` returns

**Comment by Irradiation** [comment]
THE MOP & CREMATORIUM ARE SACRED TOOLS OF MR.CLEAN.

THEY ARE TOO HOLY TO GET SINGULOTHED.

File: code/game/objects/structures/morgue.dm [hunk 1]
```diff
@@ -163,8 +163,6 @@
 
 /obj/structure/morgue/Destroy()
 	. = ..()
-	if(connected)
```
Comment (DamianX) [inline code/game/objects/structures/morgue.dm]: why did you remove this? ~~shouldn't `connected = null` be added instead?~~
you can remove the definition entirely otherwise

**Comment by Durgen** [comment]
hey github lurker spessman

it's up to you to save mops and crematoriums from the singularity

1 downthumb = 1 prayer
